libs/libc/machine/arm/gnu/arch_setjmp.S

```diff
@@ -59,7 +61,12 @@
  *
  ****************************************************************************/
 
+#ifndef __ghs__
 	.type	setjmp, function
+#else
+	.type	setjmp, $function
```

Comment:
ditto

Reply:
> ditto

same as the previous review comment, I have tried the solution, but cannot work, and report error